07_errors/patrickmarabeas/types.go

```diff
@@ -1,21 +1,19 @@
 package main
 
-import (
-	"sync"
-)
+import "sync"
 
 type Puppy struct {
 	ID    int
 	Breed string
 	Color string
-	Value string
+	Value int // cents
 }
 
 type Storer interface {
-	Create(puppy Puppy) int
-	Read(ID int) Puppy
-	Update(ID int, puppy Puppy) bool
-	Destroy(ID int) bool
+	Create(puppy Puppy) (int, error)
```

Comment:
This commit on its own breaks the code. If you run `go test ./...` on this commit, the tests will fail  - the build will fail too. This is bad as it breaks `git bisect` - you want every commit to build and pass tests.

The most minimal version of this change would be to change the implementation to always return `nil` for the error when implementing it, and a subsequent commit can start returning non-nil errors where appropriate.